src/cli/build_command.zig

```diff
@@ -386,12 +379,24 @@ pub const BuildCommand = struct {
 
                         Output.flush();
 
+                        const is_cross_compile = !compile_target.isDefault();
+
+                        if (outfile.len == 0 or strings.eqlComptime(outfile, ".") or strings.eqlComptime(outfile, "..") or strings.eqlComptime(outfile, "../")) {
```

Comment:
is this fixing a bug? also is it possible `../` has a windows path separator?

Reply:
I saw `.` happen once or twice. I did not see the other cases